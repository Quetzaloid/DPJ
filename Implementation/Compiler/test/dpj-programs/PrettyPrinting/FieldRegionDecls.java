/**
 * Basic field region declarations
 * 
 * @author Jeff Overbey;
 * @author Rob Bocchino
 */
public class FieldRegionDecls {
    region r1, r2;
    region r3;
}
