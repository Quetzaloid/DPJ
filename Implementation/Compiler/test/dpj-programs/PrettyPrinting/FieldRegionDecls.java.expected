
public class FieldRegionDecls {
    
    public FieldRegionDecls() {
        super();
    }
    region r1;
    region r2;
    region r3;
}